openslides/motions/serializers.py

```diff
@@ -2,6 +2,7 @@
 
 from django.db import transaction
 from django.utils.translation import ugettext as _
+from rest_framework import serializers
```

Comment:
Please use ValidationError from ..utils.rest_api and do not import something directly from rest_framework.

Reply:
Done.